README.md

```diff
@@ -125,6 +125,155 @@ record = client.write(record_type, data)
 print 'Wrote record {0}'.format(record.meta.record_id)
 ```
 
+## Searching records
+
```

Comment:
One of the customer requests that drove the need for v2 search was a desire for wanting a search feature that was intuitive/documented in terms of how to write records in order to be able to search on them. We should include a few snippets here in writing records with values that we search on below.